### ISSUE TITLE
nuclear_contamination + MultiQC

### ISSUE THREAD

**Issue by alexandregilardet** [nf-core/eager#714]
Hi!

## Description of the bug

MultiQC breaks on an edge case of nuclear_contamination because of the format of the standard error '6.206335-17'

## Log files

- [ ] The command used to run the pipeline
- [ ] The `.nextflow.log` file 
- [x] The exact error:
  ```
  [INFO   ]         multiqc : This is MultiQC v1.10
  [INFO   ]         multiqc : Template    : default
  [INFO   ]         multiqc : Searching   : multiqc_config.yaml
  [INFO   ]         multiqc : Searching   : /camp/lab/skoglundp/working/alexandre/results/210326_K00102_0569_BHLFGTBBXY/work/07/6584aa2bd30da57542475e4b7ff11e
  [INFO   ]         multiqc : Only using modules adapterRemoval, bowtie2, custom_content, damageprofiler, dedup, fastp, fastqc, gatk, kraken, malt, mtnucratio, multivcfanalyzer, picard, preseq, qualimap, samtools, sexdeterrmine, hops
  [INFO   ]          fastqc : Found 318 reports
  [INFO   ]  adapterRemoval : Found 159 reports
  [INFO   ]          fastqc : Found 159 reports
  [INFO   ]          kraken : Found 159 reports
  [INFO   ]        samtools : Found 159 flagstat reports
  [INFO   ]        samtools : Found 159 flagstat reports
  [INFO   ]           dedup : Found 159 reports
  [INFO   ]      mtnucratio : Found 159 reports
  [INFO   ]  custom_content : software_versions: Found 1 sample (html)
  [INFO   ]  custom_content : nf-core-eager-summary: Found 1 sample (html)
  [INFO   ]  custom_content : endorSpy: Found 159 General Statistics columns
  [INFO   ]  custom_content : nuclear_contamination: Found 159 General Statistics columns
  Traceback (most recent call last):
    File "/opt/conda/envs/nf-core-eager-2.3.2/bin/multiqc", line 6, in <module>
      from multiqc.__main__ import multiqc
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/multiqc/__main__.py", line 53, in <module>
      multiqc.run_cli(prog_name="multiqc")
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/click/core.py", line 829, in __call__
      return self.main(*args, **kwargs)
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/click/core.py", line 782, in main
      rv = self.invoke(ctx)
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
      return ctx.invoke(self.callback, **ctx.params)
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/click/core.py", line 610, in invoke
      return callback(*args, **kwargs)
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/multiqc/multiqc.py", line 248, in run_cli
      kwargs=kwargs,
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/multiqc/multiqc.py", line 728, in run
      report.general_stats_html = table.plot(report.general_stats_data, report.general_stats_headers, pconfig)
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/multiqc/plots/table.py", line 53, in plot
      return make_table(dt)
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/multiqc/plots/table.py", line 138, in make_table
      c_scale = mqc_colour.mqc_colour_scale(header["scale"], header["dmin"], header["dmax"])
    File "/opt/conda/envs/nf-core-eager-2.3.2/lib/python3.7/site-packages/multiqc/utils/mqc_colour.py", line 33, in __init__
      if float(minval) == float(maxval):
  ValueError: could not convert string to float: '6.206335-17'
  ```

## System

- Hardware: HPC
- Executor: slurm

## Nextflow Installation

- Version: 20.07.1

## Container engine

- Engine: Singularity
- version: 3.4.2
- Image tag: nfcore/eager:2.3.2

## Additional context

Removing run_nuclear_contamination allowed MultiQC to complete successfully.

**Comment by maxibor** [comment]
Seems like something @jfy133 and @TCLamnidis were dealing with recently

**Comment by maxibor** [comment]
See https://github.com/ewels/MultiQC/issues/1394

**Comment by ewels** [comment]
Fixed in MultiQC v1.10.1, updated in #715

**Comment by apeltzer** [comment]
Fixed in #715  - `dev` should already work fine now once the container has been built (~1h max). Otherwise, wait for patch release 2.3.3 coming soon ™️
